cytominer_database/utils.py

```diff
@@ -154,3 +155,61 @@ def read_config(filename):
             logger.warning("Unable to read configuration file: {}.".format(config_filename))
 
     return config
+
+
+def type_convert_dataframe(dataframe, config_file):
+    """
+    Type casting of entire pandas dataframe.
+    Calls conversion function based on specifications in configuration file.
+    :param dataframe: input file
+    :config_file: parsed configuration data (output from cytominer_database.utils.read_config(config_path))
+    """
+    engine = config_file["ingestion_engine"]["engine"]
+    if engine == "Parquet": # convert. (Else: do nothing.)
+        type_conversion = config_file["schema"]["type_conversion"]
+        if type_conversion == "int2float":
+            convert_cols_int2float(dataframe)
+        elif type_conversion == "all2string":
```

Comment:
we should do one of two things here:

1. Error this function out if the user provides something other than `int2float` or `all2string`
2. Throw a warning that this function failed to execute

We want to avoid the scenario where a user adds an unsupported option and then thinks something is converting when in reality it is not.

Reply:
That's a good point. For now I added the following: 

1. *ValueError* for a) a wrong 'type_conversion' specification and for b) failed Pandas dataframe type casting `pandas_df[name].astype("float")`or `pandas_df[name].astype("str")`
2. *UserWarning* if no conversion took place (checked by a boolean flag)